src/Server/Workspace.php

```diff
@@ -84,11 +84,11 @@ public function symbol(string $query): Promise
     {
         return coroutine(function () use ($query) {
             // Wait until indexing for definitions finished
-            if (!$this->index->isStaticComplete()) {
-                yield waitForEvent($this->index, 'static-complete');
+            if (!$this->sourceIndex->isStaticComplete()) {
```

Comment:
@jens1o its not just a rename. in some spots its now using `sourceIndex` in others `projectIndex`